usb/esp_tinyusb/tinyusb.c

```diff
@@ -92,9 +95,24 @@ esp_err_t tinyusb_driver_install(const tinyusb_config_t *config)
     tinyusb_set_descriptor(dev_descriptor, string_descriptor, string_descriptor_count, cfg_descriptor);
 
     ESP_RETURN_ON_FALSE(tusb_init(), ESP_FAIL, TAG, "Init TinyUSB stack failed");
-#if !CONFIG_TINYUSB_NO_DEFAULT_TASK
-    ESP_RETURN_ON_ERROR(tusb_run_task(), TAG, "Run TinyUSB task failed");
-#endif
+    if (!config->no_backround_task) {
+        if (config->stack_size != 0) {
+            stack_size = config->stack_size;
+        } else {
+            stack_size = 4096; // Backward compatibility with esp_tinyusb v1.0.0. Do NOT remove!
+        }
+        if (config->task_priority != 0) {
+            task_priority = config->task_priority;
+        } else {
+            task_priority = 5; // Backward compatibility with esp_tinyusb v1.0.0. Do NOT remove!
+        }
+        if (config->core_id >= 0 && (config->core_id <= configNUM_CORES - 1)) {
+            core_id = config->core_id;
+        } else {
+            core_id = tskNO_AFFINITY; // Backward compatibility with esp_tinyusb v1.0.0. Do NOT remove!
+        }
```

Comment:
Actually, we want to pin the task to core 0 by default. There are issues when the task is not pinned, so before we fix it, the task should be pinned be default.

Reply:
By default, config->core_id is 0 and task will be pinned to core 0. So, the implementation is correct. Only line to change is:
`core_id = tskNO_AFFINITY; // Backward compatibility with esp_tinyusb v1.0.0. Do NOT remove!`
->
`core_id = tskNO_AFFINITY;`

is it OK ?